portfolio/src/main/webapp/index.html

```
@@ -11,48 +11,69 @@
                     I have lived most of my life in Maryland and I love the DC Metro area <strong> | </strong>
                     I enjoy coding, writing, video editing, and going to new places with my friends <strong> | </strong>
-                    I also have recently gotten into pc building, I have three laptops and I plan on building my first desktop this summer 
+                    I also have recently gotten into PC building, I have three laptops and I plan on building my first desktop this summer. 
                 </p>
+            </div>
+        
         </div>
-     
-    </div>
 
-    <div id="tv">
-        <img src="/images/tv.jpg">
-        <div id="quote-text">
-            <div id ="quote-prompt">
-                <p><strong>Get A Random Quote From One of the Shows I Like!</strong></p>
+        <div id="hang">
+        
+            <img src="/images/HangingOut.jpg"/>
+        </div>
+
+        <div id="tv">
+            <img src="/images/tv.jpg">
+            <div id="quote-text">
+                <div id ="quote-prompt">
+                    <p><strong>Get A Random Quote From One of the Shows I Like!</strong></p>
+                </div>
+
+                <button class = "push_button red" onclick="addRandomQuote()">Click Me!</button>
+                <div id="quote-container"></div>
             </div>
 
-            <button class = "push_button red" onclick="addRandomQuote()">Click Me!</button>
-            <div id="quote-container"></div>
         </div>
 
+        <div id = "comments-section">
+            <form action = '/data' method = "POST">
+
+            <h1>Comments Section</h1>
+            <br/>
+
+            <textarea name = "comments-input">Enter your comment here!</textarea>
+            <input type = "submit">
+            </form>
+            <form action = '/data' method = "POST">
+                <h1>Pick the number of comments you want</h1>
```

You can probably mention here (if we are swallowing the exception) that 5 is default value if something goes wrong.